stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelListView.kt

```diff
@@ -26,16 +26,19 @@ public class ChannelListView @JvmOverloads constructor(
     defStyle: Int = 0
 ) : RecyclerView(context, attrs, defStyle) {
 
-    private var endReachedListener: EndReachedListener? = null
+    private val channelListItemAdapter: ChannelListItemAdapter
```

Comment:
Why do we need to store one extra field? We can reuse just `adapter` for the same purpose, can't we?

Reply:
There's a very minor difference, `BaseAdapter` not only provides items that were set to the adapter, but also pending items in async differ.

Reply:
So I felt, even though there is a very small chance chance that we invoke the method after we set channels but before async diff completes, it seems like the right thing to do.

Reply:
Ok, seems that I'm wrong here. Fixing it...